[2021-03-22 21:15:06,266] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_2 2021-03-21T00:00:00+00:00 [queued]>
[2021-03-22 21:15:06,280] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_2 2021-03-21T00:00:00+00:00 [queued]>
[2021-03-22 21:15:06,280] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-22 21:15:06,281] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-22 21:15:06,281] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-22 21:15:06,288] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): runme_2> on 2021-03-21T00:00:00+00:00
[2021-03-22 21:15:06,291] {standard_task_runner.py:52} INFO - Started process 63 to run task
[2021-03-22 21:15:06,294] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'runme_2', '2021-03-21T00:00:00+00:00', '--job-id', '8', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmp0hbk3tjo', '--error-file', '/tmp/tmpzfrpbuz0']
[2021-03-22 21:15:06,295] {standard_task_runner.py:77} INFO - Job 8: Subtask runme_2
[2021-03-22 21:15:06,334] {logging_mixin.py:104} INFO - Running <TaskInstance: example_bash_operator.runme_2 2021-03-21T00:00:00+00:00 [running]> on host 699fbf69fb73
[2021-03-22 21:15:06,369] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=runme_2
AIRFLOW_CTX_EXECUTION_DATE=2021-03-21T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-03-21T00:00:00+00:00
[2021-03-22 21:15:06,370] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-03-22 21:15:06,371] {bash.py:158} INFO - Running command: echo "example_bash_operator__runme_2__20210321" && sleep 1
[2021-03-22 21:15:06,377] {bash.py:169} INFO - Output:
[2021-03-22 21:15:06,378] {bash.py:173} INFO - example_bash_operator__runme_2__20210321
[2021-03-22 21:15:07,379] {bash.py:177} INFO - Command exited with return code 0
[2021-03-22 21:15:07,401] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_2, execution_date=20210321T000000, start_date=20210322T211506, end_date=20210322T211507
[2021-03-22 21:15:07,426] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-03-22 21:15:07,431] {local_task_job.py:146} INFO - Task exited with return code 0
